src/Sentry/Internal/ProcessInfo.cs

```diff
@@ -25,6 +26,24 @@ internal Task PreciseAppStartupTask
         private set => _preciseAppStartupTask = value;
     }
 
+    public int? Id => _id ??= GetCurrentProcessId();
```

Comment:
This code path can end up on the P/Invoke below expecting `user32.dll`and will crash outside Windows.
We need to have runtime OS check before running that stuff.

Also, CI is green which indicates to me we don't have code calling this, need some tests.

Reply:
`GetCurrentProcessId` doesn't use PInvoke I believe.